### ISSUE TITLE
Driver installation Issue

### ISSUE THREAD

**Issue by ganmahmud** [intekhabrizvi/codeigniter-mongodb-session-driver#3]
@intekhabrizvi 
I have put the Session_mongo_driver in `application > libraries > Session > drivers` and my config.php is as follows -
```
$config['sess_driver'] = 'mongo';
$config['sess_cookie_name'] = 'ci_session';
$config['sess_expiration'] = 7200;
$config['sess_save_path'] = 'mongodb://user_id:password@localhost:27017/database|database|collection';
$config['sess_match_ip'] = TRUE;
$config['sess_time_to_update'] = 300;
$config['sess_regenerate_destroy'] = FALSE;
```
I am getting the following error:
```
Message: Session: Configured driver 'mongo' was not found. Aborting.
```
What am I doing wrong here ?
note: I am using CI version 3.

**Comment by intekhabrizvi** [comment]
Fixed applied in 1ce4999 commit, can you pls try.

**Comment by ganmahmud** [comment]
Sweet. Working perfectly now. Thanks a lot for the fix.
